hyperspy/__init__.py

```diff
@@ -39,7 +26,6 @@
 
 import Release
 
+__all__ = ["api"]
```

Comment:
I'm unsure whether this is necessary or not. Don't we risk hiding functionality for those that want to explore? What are the pros/cons?